dll/win32/browseui/resource.h

```diff
@@ -125,6 +125,7 @@
 #define IDS_STANDARD_TOOLBAR     12624
 #define IDS_ADDRESSBANDLABEL     12902
 #define IDS_FOLDERSLABEL         12919
+#define IDS_ADDRESSMENUTEXT      13137
```

Comment:
A value in quite of different range from those of its surroundings! (e.g. the `IDS_HISTORYTEXT` one). Any reason why?

Reply:
That is from `addressband.rgs`.